### PR TITLE
Add Raceid workflow

### PR DESCRIPTION
**Pull request by mtekman**

### CODE REVIEW

File: raceid3/raceid3_workflow-test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+- doc: Test RaceID workflow in Galaxy Europe
+  job:
+    count_matrix:
+      class: File
+      location: use.intestinal
```
Comment (bgruening) [inline raceid3/raceid3_workflow-test.yml]: Should that not be a URL pointing to a real matrix?

Comment (mtekman) [reply raceid3/raceid3_workflow-test.yml]: The file is literally a 1x1 'matrix' with the word "test". It is a signature file of sorts, that tells the tool to  load the internal example data. 

I could upload the raw example data to zenodo (~30MB) if that would be better.

Comment (bgruening) [reply raceid3/raceid3_workflow-test.yml]: @mtekman please do not use this `load internal data`, this is a huge hack. A matrix that can be potentially used in the training material would make a lot of sense here. We want to test a real workflow here and not the fake hack, imho.

Comment (mtekman) [reply raceid3/raceid3_workflow-test.yml]: @bgruening fair enough, will upload the dataset to zenodo

Comment (bgruening) [reply raceid3/raceid3_workflow-test.yml]: Thanks.

Comment (mtekman) [reply raceid3/raceid3_workflow-test.yml]: @bgruening tests run good, any further changes?

**Comment by bgruening** [comment]
Cool, let's see :)

**Comment by bgruening** [comment]
Do you also have a test workflow for the pre-processing?

**Comment by mtekman** [comment]
@bgruening am working on it

**Pull request merged**